### PR TITLE
Add rule for 4.2.7

### PR DESCRIPTION
**Pull request by vakwetu**
#### Description:

- Add rule for 4.2.7

### CODE REVIEW

File: ocp4/profiles/cis-node.profile [hunk 1]
```diff
@@ -88,6 +88,7 @@ selections:
     # FIXME(jhrozek): This does not seem to be set in OCP explicitly and the code seems to suggest
     # that the default is false? Need to confirm
   # 4.2.7 Ensure that the --make-iptables-util-chains argument is set to true
+    - kubelet_enable_iptables_util_chains
     # - like kubelet_anonymous_auth_disabled but check for makeIPTablesUtilChains is NOT set to false (true is the default)
```
Comment (nkinder) [inline ocp4/profiles/cis-node.profile:92]: (nit) We should probably remove the comment about how to implement the rule.

**Comment by redhatrises** [comment]
I am going to merge this. There will definitely be a clean up PR to address nitpicks, typos, duplicate rules, etc.

**Pull request merged**